src/tests/unit/test_metric_registry.py

```
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""
+Comprehensive unit tests for the metric registry system.
+Tests the registration, retrieval, and management of fairness metrics.
+"""
+
+import unittest
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)